### PR TITLE
add scheduled class to student

### PR DESCRIPTION
**Pull request by lewisorenge**
add scheduled class to student

### CODE REVIEW

File: apps/classrooms/migrations/0007_auto_20201012_1800.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('classrooms', '0006_auto_20200925_1008'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```
Comment (antorenge) [inline apps/classrooms/migrations/0007_auto_20201012_1800.py:13]: Why this change?

File: apps/schools/api/views.py [hunk 2]
```diff
@@ -412,6 +412,9 @@ def create_classroom_view(request, *args, **kwargs):
         event.save()
         # Add scheduled class to teachers calendar
         event.calendars.add(course.teacher.user.calendar)
+        # Add scheduled class to students calender
+        for student in course.students.all():
+            event.calendars.add(calendars.Calendar.objects.get_calendar_for_object(student.user))
```
Comment (antorenge) [inline apps/schools/api/views.py:417]: Why not just do `student.user.calendar` ?

File: apps/schools/migrations/0007_auto_20210129_1715.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('schools', '0006_teacher_position'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```
Comment (antorenge) [inline apps/schools/migrations/0007_auto_20210129_1715.py:13]: Why this change?

**Review by antorenge — CHANGES_REQUESTED** [review]

File: apps/classrooms/migrations/0007_auto_20201012_1800.py (hunk 1 above)
Comment (lewisorenge) [reply apps/classrooms/migrations/0007_auto_20201012_1800.py:13]: I did not add to migrations to git ignore

File: apps/schools/migrations/0007_auto_20210129_1715.py (hunk 3 above)
Comment (lewisorenge) [reply apps/schools/migrations/0007_auto_20210129_1715.py:13]: I did not add migrations to git ignore

File: apps/schools/api/views.py (hunk 2 above)
Comment (lewisorenge) [reply apps/schools/api/views.py:417]: Tried doing it that  way on the first try i got an attribute error "the user object  had no object calender" which thats why iopted for that path.

File: apps/classrooms/migrations/0007_auto_20201012_1800.py (hunk 1 above)
Comment (antorenge) [reply apps/classrooms/migrations/0007_auto_20201012_1800.py:13]: Migrations should be included in our workflow

File: apps/schools/api/views.py (hunk 2 above)
Comment (antorenge) [reply apps/schools/api/views.py:417]: Both approaches will achieve the same objective. However, the cleaner one should be the `student.user.calendar`. You might want to debug why the user is missing the calendar object at that point.

**Review by antorenge — COMMENTED** [review]
Tests are also missing.

File: apps/schools/api/views.py (hunk 2 above)
Comment (lewisorenge) [reply apps/schools/api/views.py]: Yeah, it was a typo i was using 'calender' instead of 'calendar'. I have updated it.

File: .gitignore [hunk 4]
```diff
@@ -84,6 +84,11 @@ ENV/
 env.bak/
 venv.bak/
 
+# Migrations Files
```
Comment (antorenge) [inline .gitignore:87]: Don't ignore migrations - we need them!

File: apps/schools/api/views.py [hunk 5]
```diff
@@ -412,6 +412,9 @@ def create_classroom_view(request, *args, **kwargs):
         event.save()
         # Add scheduled class to teachers calendar
         event.calendars.add(course.teacher.user.calendar)
+        # Add scheduled class to students calender
```
Comment (antorenge) [inline apps/schools/api/views.py:415]: Add a test to cover this functionality. See https://www.django-rest-framework.org/api-guide/testing/ or https://docs.djangoproject.com/en/3.1/topics/testing/

**Review by antorenge — CHANGES_REQUESTED** [review]
